src/structs/Webhook.ts

```diff
@@ -0,0 +1,98 @@
+import { EventEmitter } from 'events'
+import getBody from 'raw-body'
+import qs from 'querystring'
+
+export interface Webhook {
+  on(event: 'vote', listener: (vote: Vote) => void): this
+}
+
+/**
+ * Top.gg Webhook
+ * @extends EventEmitter
+ * @example
+ * ```js
+ * const express = require('express')
+ * const { Webhook } = require('@top-gg/sdk')
+ * 
+ * const app = express()
+ * const wh = new Webhook('webhookauth123')
+ * 
+ * app.post('/dblwebhook', wh.middleware())
+ * 
+ * wh.on('vote', (vote) => {
+ *   vote.user
+ *   // => 321714991050784770
+ * })
+ * 
```

Comment:
If we have middleware for this, it kind of defeats the point of having an event emitter. I would expect a middleware based API to look like this instead.
```js
const { Api } = require('@top-gg/sdk')

const topgg = new Api(...);

app.post('/dblwebhook', topgg.middleware('webhookauth123'), (req, res) => {
  console.log(req.vote); // vote attached to the request object
});